dts/bindings/sensor/st,lps22hh-common.yaml

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2021 STMicroelectronics
+# SPDX-License-Identifier: Apache-2.0
+
+properties:
+    drdy-gpios:
+      type: phandle-array
+      required: false
+      description: DRDY pin
+
+        This pin defaults to active high when produced by the sensor.
+        The property value should ensure the flags properly describe
+        the signal that is presented to the driver.
```

Comment:
multiline properties need to be described using:

```yaml
description: |
     the details
```

Reply:
Yes, sorry. I just moved this property in the commoin file, but I forgot to change it. Thanks for catching!